recipes/pixi-inspect/recipe.yaml

```
@@ -0,0 +1,49 @@
+context:
+  name: pixi-inspect
+  version: "2.0.2"
+  sha256: "039d989c48764a0c6c63ee1d8c9ca6e8cd2d12e05ef00e8a04c9a5a1cadefbe7"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/toprinse/pixi-inspect/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: ${{ sha256 }}
```

There is no need for this indirection
```suggestion
  sha256: 039d989c48764a0c6c63ee1d8c9ca6e8cd2d12e05ef00e8a04c9a5a1cadefbe7
```